test/dynamo/test_profiler.py

```diff
@@ -176,7 +176,21 @@ def fn(x, y, z):
         with torch.profiler.profile() as prof:
             opt_fn(*inputs)
 
-        self.assertTrue(any(e.name == "Torch-Compiled Region" for e in prof.events()))
+        for e in prof.events():
+            if "Torch-Compiled Region" in e.name:
+                print(e.name)
```

Comment:
don't forget to remove this

Reply:
Removed, forgot that in OSS it will print it even on pass.